packages/fxa-settings/src/components/PageDeleteAccount/en-US.ftl

```diff
@@ -6,7 +6,7 @@ delete-account-header =
 delete-account-step-1-2 = Step 1 of 2
 delete-account-step-2-2 = Step 2 of 2
 
-delete-account-confirm-title-2 = You’ve connected your { -product-firefox-account } to { -brand-mozilla } products that keep you secure and productive on the web:
+delete-account-confirm-title-2 = You’ve connected your { -product-firefox-account } to { -brand-mozilla } products that keep you secure and productive on the web.
```

Comment:
I guess we will also need a new string version for this :/

Reply:
Like, the translators will need to generate new translations for this string?

Reply:
The comment above means: the string changed, we need to signal localizers if we want them to pick the change and be consistent with English.

More info [here](https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html).

Please update the string ID in a follow-up ;-)

Reply:
I filed https://mozilla-hub.atlassian.net/browse/FXA-5049 and put it in our current sprint

Reply:
thank you! I'll get it updated today!

Reply:
done!